content/blog/2025-05-27-welcoming-almalinux-10.md

```diff
@@ -0,0 +1,111 @@
+AlmaLinux OS 10.0 also includes a tech-preview of KVM virtualization support for the IBM Power architecture. It has been unavailable upstream since version 9.0, but is fundamental for a number of AlmaLinux users. That list includes the [Oregon State University Open Source Lab](https://osuosl.org/), who submitted the [RFC to the AlmaLinux Engineering Steering Committee](https://github.com/AlmaLinux/ALESCo/blob/master/rfcs/0002-enable-kvm-on-almaLinux-9-on-ppc64le.md) for consideration in February.
+
+### Continuing our expanded hardware support
+Starting with AlmaLinux 8.10 and 9.4 we re-enabled support for more than 150 devices that were removed upstream. Those additions continue in AlmaLinux OS 10.0. You can see the full list of devices in the [AlmaLinux OS 10.0 release notes](https://wiki.almalinux.org/release-notes/10.0.html).
+
+## ISOs, Live Images, Cloud and Containers
+
+AlmaLinux also offers a variety of Cloud, Container and Live Images. The builds for these get kicked off as soon as the public repository is ready. 
+
+Installation ISOs are available on the mirrors now for all architectures:
+* [Intel/AMD (x86-64)](https://mirrors.almalinux.org/isos/x86_64/10.0.html)
+* [Intel/AMD (x86-64-v2)](https://mirrors.almalinux.org/isos/x86_64_v2/10.0.html)
+* [ARM64 (aarch64)](https://mirrors.almalinux.org/isos/aarch64/10.0.html)
+* [IBM PowerPC (ppc64le)](https://mirrors.almalinux.org/isos/ppc64le/10.0.html)
+* [IBM Z (s390x)](https://mirrors.almalinux.org/isos/s390x/10.0.html)
+
+Torrents are available as well at:
+* [Intel/AMD (x86-64)](https://repo.almalinux.org/almalinux/10.0/isos/x86_64/AlmaLinux-10.0-x86_64.torrent)
+* [Intel/AMD (x86-64-v2)](https://repo.almalinux.org/almalinux/10.0/isos/x86_64_v2/AlmaLinux-10.0-x86_64_v2.torrent)
+* [ARM64 (aarch64)](https://repo.almalinux.org/almalinux/10.0/isos/aarch64/AlmaLinux-10.0-aarch64.torrent)
+* [IBM PowerPC (ppc64le)](https://repo.almalinux.org/almalinux/10.0/isos/ppc64le/AlmaLinux-10.0-ppc64le.torrent)
+* [IBM Z (s390x)](https://repo.almalinux.org/almalinux/10.0/isos/s390x/AlmaLinux-10.0-s390x.torrent)
+
+**The following images are expected to be available shortly.** 
+
+* [Container](https://wiki.almalinux.org/containers/) images including Platform and UBIs alternatives. We provide a wide variety of containers for your use. 
+* [LXC/LXD](https://images.linuxcontainers.org/images/almalinux/) 
+* [Live Media](https://wiki.almalinux.org/LiveMedia.html) for GNOME, GNOME-mini, KDE.
+* Cloud Images 
+    * [AWS](https://wiki.almalinux.org/cloud/AWS.html) for x86_64 and AArch64 EC2 Instances
+    * [Azure](https://wiki.almalinux.org/cloud/Azure.html) for x86_64 and AArch64 VMs
+    * [Google Cloud](https://wiki.almalinux.org/cloud/Google.html)
+    * [Oracle Cloud](https://wiki.almalinux.org/cloud/OCI.html) for x86_64 and AArch64 VMs
+    * [Generic Cloud/Cloud-init](https://wiki.almalinux.org/cloud/Generic-cloud-on-local.html) for all supported architectures including x86-64-v2
```

Comment:
- Could you update the name of the image from [Generic Cloud/Cloud-init] to Generic Cloud with Cloud-init (OpenStack compatible)
- Could you update the page of Generic Cloud image?
https://wiki.almalinux.org/cloud/Generic-cloud.html

Reply:
Submit a proposal for what you'd like.